back/identify/LinkedIn/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+# Create your models here.
+
+
+class LinkedIn(models.Model):
```

Comment:
Why do all fields have null True?
We need some fields to be required
Like first and last name for instance
First and last name should not have null True